auth/basics/js.md

```diff
@@ -93,15 +93,15 @@ skygear.auth.signupWithUsername(username, password).then((user) => {
   console.log(user); // user object
 }, (error) => {
   console.error(error);
-  if (error.error.code === skygearError.Duplicated) {
+  if (error.error.code === skygear.ErrorCodes.Duplicated) {
```

Comment:
If `skygear.ErrorCodes` is the correct way to get the error codes, then I guess `import skygearError from 'skygear/lib/error';` is not needed in the previous lines.

Same for other error codes example code update.